### PR TITLE
Update supported plugin contexts

### PR DESCRIPTION
**Pull request by dfreier**

### CODE REVIEW

File: content/reference/document/metadata/plugins/li-reference.md [hunk 1]
```diff
@@ -9,8 +9,8 @@ summary: |
 support:
   document: true
   media: true
-  tableDashboard: false
   include: false
+  tableDashboard: false
```
Comment (ajwild) [inline content/reference/document/metadata/plugins/li-reference.md:13]: `true`

File: content/reference/document/metadata/plugins/li-reference.md [hunk 2]
```diff
@@ -9,8 +9,8 @@ summary: |
 support:
   document: true
   media: true
-  tableDashboard: false
   include: false
```
Comment (ajwild) [inline content/reference/document/metadata/plugins/li-reference.md:12]: `true`

File: content/reference/document/metadata/plugins/li-reference-list.md [hunk 3]
```diff
@@ -9,8 +9,8 @@ summary: |
 support:
   document: true
   media: true
-  tableDashboard: false
   include: false
```
Comment (ajwild) [inline content/reference/document/metadata/plugins/li-reference-list.md:12]: `true`

File: content/reference/document/metadata/plugins/li-team.md [hunk 4]
```diff
@@ -8,8 +8,10 @@ summary: Display team in metadata form or dashboard.
 support:
   document: true
   media: false
-  tableDashboard: true
   include: false
+  creationFlow: true
+  pushMessage: false
+  tableDashboard: true
   displayFilter: false
```
Comment (ajwild) [inline content/reference/document/metadata/plugins/li-team.md:15]: `true`

**Review by ajwild — APPROVED** [review]
Thanks for tidying it up. I reviewed the other `supported` properties as well and found a few issues. If you have time please could you update your commit, otherwise I can open a new PR.

While the `li-reference` and `li-reference-list` plugins are no longer in the code I think it's worth having the correct values for when the did exist so that customers who are a couple of releases behind can still have an accurate reference.

File: content/reference/document/metadata/plugins/li-routing.md [hunk 5]
```diff
@@ -8,8 +8,10 @@ summary: Define aliases.
 support:
   document: true
   media: false
-  tableDashboard: false
   include: false
+  creationFlow: false
+  pushMessage: false
+  tableDashboard: false
   displayFilter: false
   searchIndexing: false
```
[the remark this answers is not in the document]
Comment (dfreier) [reply content/reference/document/metadata/plugins/li-routing.md:16]: You mean `searchIndexing: true`?

File: content/reference/document/metadata/plugins/li-task-v2.md [hunk 6]
```diff
@@ -7,8 +7,10 @@ menus:
 support:
   document: true
   media: false
-  tableDashboard: false
   include: false
+  creationFlow: false
+  pushMessage: false
+  tableDashboard: false
```
[the remark this answers is not in the document]
Comment (dfreier) [reply content/reference/document/metadata/plugins/li-task-v2.md:13]: You mean `tableDashboard: true`?

File: content/reference/document/metadata/plugins/li-reference.md (hunk 2 above)
Comment (dfreier) [reply content/reference/document/metadata/plugins/li-reference.md:12]: You mean `tableDashboard: true`

File: content/reference/document/metadata/plugins/li-reference-list.md (hunk 3 above)
Comment (dfreier) [reply content/reference/document/metadata/plugins/li-reference-list.md:12]: You mean `tableDashboard: true`?

File: content/reference/document/metadata/plugins/li-team.md (hunk 4 above)
Comment (dfreier) [reply content/reference/document/metadata/plugins/li-team.md:15]: You mean `displayFilter : true`?

**Comment by ajwild** [comment]
@dfreier sorry that it's not so clear. It's always the last line that you can see. I thought the single line highlight was more obvious :see_no_evil:

File: content/reference/document/metadata/plugins/li-reference.md (hunk 2 above)
Comment (ajwild) [reply content/reference/document/metadata/plugins/li-reference.md:12]: This one is `include`.

File: content/reference/document/metadata/plugins/li-reference-list.md (hunk 3 above)
Comment (ajwild) [reply content/reference/document/metadata/plugins/li-reference-list.md:12]: This one is `include`.

Comment (dfreier) [reply content/reference/document/metadata/plugins/li-reference-list.md:12]: So include?

File: content/reference/document/metadata/plugins/li-reference.md (hunk 2 above)
Comment (dfreier) [reply content/reference/document/metadata/plugins/li-reference.md:12]: So include?

**Comment by dfreier** [comment]
The single-line highlighting really has some room for improvement on GH 😄

**Pull request merged**